### ISSUE TITLE
Build failing because of okhttp3 mismatch

### ISSUE THREAD

**Issue by spencergibb** [spring-cloud/spring-cloud-kubernetes#397]
```
Exception in thread "MockWebServer" java.lang.NoSuchFieldError: H2_PRIOR_KNOWLEDGE
	at okhttp3.mockwebserver.MockWebServer$4.processConnection(MockWebServer.java:513)
	at okhttp3.mockwebserver.MockWebServer$4.execute(MockWebServer.java:456)
	at okhttp3.internal.NamedRunnable.run(NamedRunnable.java:32)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by geoand** [comment]
I'll take a look

**Comment by geoand** [comment]
So this only happens in `master`, `1.0.x` works fine.

**Comment by geoand** [comment]
Fixed by #398
